src/njoy21/input/THERMR/Card2/Nbin.hpp

```diff
@@ -0,0 +1,18 @@
+struct Nbin {
+  using Value_t = int;
+  static std::string name(){ return "nbin"; }
+
+  static std::string description(){
+    return
+      "nbin is the number of equi-probable scattering angles available. For\n"
```

Comment:
That's reasonable, I was honestly very unsure of how best to word it, since I myself was having some difficulty understanding it. That said, does this look any better? 
```
     "nbin is the number of equi-probable scattering angles available. So\n"
      "the scattering distribution is divided into nbin different bins,\n"
      "where the areas within each of the bins are equal to each other.\n"
      "\n"
      "nbin must be a positive integer.";
```